velox/exec/HashTable.h

```diff
@@ -74,7 +77,8 @@ struct HashLookup {
   /// Input to groupProbe and joinProbe APIs.
 
   /// Set of row numbers of row to probe.
-  raw_vector<vector_size_t> rows;
+  raw_vector<vector_size_t, AlignedStlAllocator<vector_size_t, simd::kPadding>>
+      rows;
 
   /// Hashes or value IDs for rows in 'rows'. Not aligned with 'rows'. Index is
   /// the row number.
```

Comment:
There are bunch more raw_vector below this in HashLookup (hashes, hits, newGroups, normalizedKeys). All these are also proportional to the number of input rows in addInput(). We should use the HashStringAllocator for them.

Reply:
Right, I started with one. More changes to specify the templated type in various places would be needed. I had started on this after pushing this PR. Will continue making the changes and try them out.